.travis.yml

```diff
@@ -1,15 +1,16 @@
+---
 language: ruby
 rvm:
-- "2.1"
-- "2.2.5"
-- "2.3.1"
-- ruby-head
-- jruby-head
+  - "2.3.8"
```

Comment:
YAML usually doesn't have a leading space for an array element

Reply:
Yeah right, I followed the rules in my local vim linter... 
Changing right now

Reply:
Ah okay, yeah it looks odd to me I'd expect a leading space as well but :man_shrugging:

Reply:
No no you're right, I've checked also the yaml spec on the site... I've always to recheck... counterintuitive